kcbq-connector/src/integration-test/java/com/wepay/kafka/connect/bigquery/it/BigQueryConnectorIntegrationTest.java

```diff
@@ -107,11 +109,13 @@ private static void initializeTestProperties() throws Exception {
             + "' property must be specified in test properties file"
         );
       }
+
+      keyFileType = properties.getProperty(KEYFILE_TYPE_PROPERTY);
```

Comment:
We don't want to make this config required in tests so we will not throw an error if the config is null/empty